TypeCobol/Compiler/SqlCodeElements/Statement/CommitStatement.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
Remove all unused using directives.

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)